deps/rabbitmq_management/priv/schema/rabbitmq_management.schema

```diff
@@ -646,3 +646,10 @@ end}.
     {datatype, {enum, [true, false]}},
     {include_default, false}
 ]}.
+
+%% Block enabling certain feature flags over API
+
+{mapping, "management.restrictions.feature_flag_blocked.$name", "rabbitmq_management.restrictions.feature_flag_blocked", [
```

Comment:
What do you think about renaming this key to `management.restrictions.blocked_feature_flags.$name` (in plural with the adjective coming first, as it would in a piece of prose in English)?